docker-compose.yml

```diff
@@ -158,6 +161,7 @@ services:
     volumes:
       - ./app:/usr/src/code/app
       - ./src:/usr/src/code/src
+      - ./vendor:/usr/src/code/vendor
```

Comment:
Better to keep this as comment by default.

Reply:
Removed.